tests/sentry/lang/native/test_plugin.py

```diff
@@ -1294,8 +1294,7 @@ def test_debug_id_resolving(self):
         frames = bt.frames
 
         assert frames[0].function == 'main'
-        # NOTE: Breakpad symbols only contain entire paths
-        assert frames[0].filename == 'c:\\projects\\breakpad-tools\\windows\\crash\\main.cpp'
```

Comment:
I assume the change here is that symbolic now calculates the basename from breakpad?

Reply:
Correct.  See [here](https://github.com/getsentry/symbolic/blob/dbcb5cef96375308689bdfbaba43e981b1b4ce8c/debuginfo/src/breakpad.rs#L1082).